### ISSUE TITLE
Improve more actions button accessibility in the context menu component

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#10586]
### Problem to solve

The more actions button in the context menu component currently comes with poor accessibility.

Generally, a div-based button can become quite accessible, however, you'd still need to make the element focusable and define event handlers when a button-based button automatically takes care of this. See relevant documentation[[1](https://www.w3.org/WAI/ARIA/apg/patterns/button/)][[2](https://www.w3.org/WAI/ARIA/apg/example-index/button/button.html)][[3](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)][...].

For example, in this case where we're using a div-based button, we're missing the basic `role="button"` attribute, which could improve the existing situation. ➿ 

This component is also used [`11 times`](https://github.com/gitpod-io/gitpod/search?q=%22ContextMenu%22%3B%22) across the codebase, most commonly within the [Items List component](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/ItemsList.tsx). 🔍

**Comment by Kanika637** [comment]
Hey @gtsiolis , I want to work on this issue, Could you guide a little bit on this.
